### PR TITLE
fozzie-components@v1.7.0 – f-skeleton-component added for creating base components

### PR DESCRIPTION
**Pull request by ashleynolan**
### Added
- `f-skeleton-component` added for use when creating base component structure.

### CODE REVIEW

File: packages/f-skeleton-component/src/assets/scss/common.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// Set $theme variable so that fozzie doesn't complain when vars are imported below
+// n.b. This variables isn't actually used (unless there are theme specific vars from fozzie)
+// Instead, the theme is handled via a data-attribute on the component (via the `@mixin theme()` below)
+$theme: 'je' !default;
+
+@import '~@justeat/f-utils/src/scss/functions/index';
```
Comment (DamianMullins) [inline packages/f-skeleton-component/src/assets/scss/common.scss:6]: Should you include `f-utils` and `fozzie` in the project dependencies?

File: packages/f-skeleton-component/src/tenants/da-DK.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export default {
+    locale: 'da-DK',
```
Comment (DamianMullins) [inline packages/f-skeleton-component/src/tenants/da-DK.js:2]: Do we need the `locale` property in these files? Aren't we already selecting the translations in code using a locale string?

File: packages/f-skeleton-component/src/assets/scss/common.scss (hunk 1 above)
Comment (ashleynolan) [reply packages/f-skeleton-component/src/assets/scss/common.scss:6]: Shouldn't need to, as these are hoisted in the mono-repo dependencies (in the base `package.json`)

File: packages/f-skeleton-component/src/tenants/da-DK.js (hunk 2 above)
Comment (ashleynolan) [reply packages/f-skeleton-component/src/tenants/da-DK.js:2]: It's a fallback for if the component is used when the `i18n` module isn't being used. In header/footer we use [this shared call](https://github.com/justeat/fozzie-components/blob/master/packages/f-header/src/components/Header.vue#L104) to check which locale setting to use.

Comment (ashleynolan) [reply packages/f-skeleton-component/src/tenants/da-DK.js:2]: This is the shared service method that does that check too > https://github.com/justeat/fozzie-components/blob/master/packages/f-services/src/services/shared.service.js#L25-L28

**Review by DamianMullins — APPROVED** [review]

**Review by xander-marjoram — APPROVED** [review]

**Pull request merged**